javascript/ql/src/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -24,9 +27,18 @@ module NodeJSLib {
    * Gets a reference to the 'process' object.
    */
   DataFlow::SourceNode process() {
+    result = DataFlow::globalVarRef("process") or
     result = DataFlow::moduleImport("process")
   }
 
+  /**
+   * Gets a reference to a member of the 'process' object.
+   */
+  private DataFlow::SourceNode processMember(string member) {
+    result = process().getAPropertyRead(member) or
+    result = DataFlow::moduleMember("process", member)
```

Comment:
I believe the second case is redundant? Destructuring imports are now modelled as a module import with prop reads for each selector.

Reply:
Possible. I basically just reverted to what it was before https://github.com/Semmle/ql/pull/1432.

Reply:
Amended.